src/types.ts

```diff
@@ -6,6 +8,7 @@ export interface Config {
   name?: string;
   type?: string;
   scope?: string;
+  builds?: Builder[];
```

Comment:
Doesn't need to be in this PR, but we should merge this definition with `NowConfig` in `src/util/dev/types.ts`.

Reply:
I made it at least extend it instead of using the type from `@now/build-utils`.